R/scenario-configuration.R

```diff
@@ -25,9 +25,28 @@ ScenarioConfiguration <- R6::R6Class(
         private$.simulateSteadyState
       } else {
         validateIsLogical(value)
+        # If the value is `NA`, do not change
+        if (is.na(value)) {
+          invisible()
+        }
         private$.simulateSteadyState <- value
       }
     },
+    #' @field readPopulationFromCSV Boolean representing whether the a new population
+    #' will be created (value is `FALSE`) or an existing population will be imported
+    #' from a csv.
+    readPopulationFromCSV = function(value) {
```

Comment:
This is the new option that defines if the population will be created or imported from file.